Cyborg Network Milestone Delivery

beekay2706 opened the pull request:
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [ ] This pull request is being made by the same account as the accepted application.
- [ ] I have disclosed any and all sources of reused code in the submitted repositories and have done my due diligence to meet its license requirements.
- [ ] In case of acceptance, invoices must be submitted and payments will be transferred to the Polkadot AssetHub and/or fiat account provided in the application.
- [ ] The delivery is according to the [Guidelines for Milestone Deliverables](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/2261

semuelle commented on deliveries/cyborg.md:
```diff
@@ -0,0 +1,26 @@
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [PR 2261](https://github.com/w3f/Grants-Program/pull/2261)
```
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/Cyborg.md
```

ditavia-br commented:
Hi, We'll do this evaluation.

ditavia-br commented:
@beekay2706 thanks for the milestone delivery and my apologies for the delay in the evaluation. Please take a look at the evaluation document [in this PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/1201) and provide proper answers and fixed. After that, let me know to continue this evaluation.

beekay2706 commented:
Hi @ditavia-br,

We have updated the detailed information and instructions to address the issues mentioned in the evaluation. I will add some comments to your evaluation PR to clarify certain aspects.

ditavia-br commented:
@beekay2706 we tested again and had some problems. Please check the evaluation document in [this PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/1201) and provide proper answers and fixes. Let me know when we can continue this evaluation.

beekay2706 commented:
@ditavia-br

Thanks for taking the time to carry out a second evaluation. We value your feedback and the effort you put into testing this. From your Evaluation 2, it seems that some networking considerations weren't emphasized enough by us, which led to different results. We've now updated the step-by-step instructions for setting up the K3S infrastructure in Azure.

Please check the latest updates.

ditavia-br commented:
@beekay2706 thanks for the improvements and fixes. We did a new evaluation round but we still having some problems to run some parts of the application. Please check the evaluation document [in this PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/1201) and let us know when we could continue this evaluation.

beekay2706 commented:
Hi Ditavia team, 
Thank you for taking the effort to test the delivery thrice,

We really embrace the effort you put in to the details, Over the last two weeks our team has put in a diligent effort to improvise the testing suite and documentation. Here is the summary of the main changes

1.  Created a fully local testing suite using UTM 
2. Video Documentation for recreating the local testing environment
3. UI Bug Fixes
4. Fixed Connectivity issues with K3S worker
5. Fixed Issues in API calls for Logs and metrics.

Please take a look at the updates and let us know.

ditavia-br commented:
@beekay2706 thank you for the improvements and fixes. We checked again and the milestone is accepted for our side. Great work! There are some small improvements in the documentation that you can check [in the evaluation PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/1201).

beekay2706 commented:
> @beekay2706 thank you for the improvements and fixes. We checked again and the milestone is accepted for our side. Great work! There are some small improvements in the documentation that you can check [in the evaluation PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/1201).

Thank you so much, @ditavia-br ! We'll swiftly update the documentation with all the suggested improvements.

The depth of detail your team provided during the testing process has given us valuable insights, helping us as a young team to plan our development more effectively. We'll apply these learnings and demonstrate continuous improvement in future deliveries.

Even though we've gone through four evaluations, I can already see a significant improvement in the app's interface compared to where we started in the first evaluation. Thanks to your entire team!

keeganquigley reviewed (approved):
Thanks @beekay2706 for all the fixes, and for your patience during the evaluation process. The external evaluator has accepted the eval, therefore I'm happy to pass the milestone. Congrats and nice work!

RouvenP commented:
hi @beekay2706 we sent the payments today
